tests/max_config/nidaqmxMaxConfig.ini

```diff
@@ -29,8 +29,10 @@ DevSerialNum = 0x0
 DevIsSimulated = 1
 ProductNum = 0x74A9C4C4
 BusType = PXIe
-PXI.ChassisNum = 1
-PXI.SlotNum = 2
+PXI.ChassisNum = 4294967295
+PXI.SlotNum = 4294967295
```

Comment:
please revert this change. you can just do it manually, the import should still work

Reply:
Noted. I have reverted the two lines